Makefile.in

```diff
@@ -349,7 +349,7 @@ libaddon: exports
 	$(MAKE) -C lib/addons/library.kodi.inputstream
 addon-dev-kit:
 	mkdir -p xbmc/addons/kodi-addon-dev-kit/build
-	cd xbmc/addons/kodi-addon-dev-kit/build; $(CMAKE) -DCMAKE_INSTALL_PREFIX=$(DESTDIR)$(prefix) ..
+	cd xbmc/addons/kodi-addon-dev-kit/build; cmake -DCMAKE_INSTALL_PREFIX=$(DESTDIR)$(prefix) ..
```

Comment:
this doesn't make any sense. There is no "target cmake", only the one we build in depends, which is "host native", but is configured with the correct toolchain, cflags etc.
see https://github.com/xbmc/xbmc/blob/master/tools/depends/Makefile.include.in#L58
after depends configure, its Makefile.include
You must use this cmake, so $(CMAKE) is correct for cross compiling. For linux standalone(non cross), you just have to check if $(CMAKE) is empty and set it to "cmake" like you had previously in the addon-dev-kit Makefile. Or make the check for cmake in configure do this for you.